### PR TITLE
Add log to avoid warning for ineffectual assignment

### PR DESCRIPTION
**Pull request by isaachier**
Signed-off-by: Isaac Hier <ihier@uber.com>

### CODE REVIEW

File: examples/hotrod/services/driver/redis.go [hunk 1]
```diff
@@ -54,6 +54,7 @@ func (r *Redis) FindDriverIDs(ctx context.Context, location string) []string {
 		ext.SpanKindRPCClient.Set(span)
 		defer span.Finish()
 		ctx = opentracing.ContextWithSpan(ctx, span)
+		r.logger.For(ctx).Info("FindDriverIDs at " + location)
```
Comment (yurishkuro) [inline examples/hotrod/services/driver/redis.go]: that's not useful. I would rather have this just before the `return` statement and log the `drivers` array.

Comment (isaachier) [reply examples/hotrod/services/driver/redis.go]: Got it OK

File: examples/hotrod/services/driver/redis.go [hunk 2]
```diff
@@ -62,6 +63,8 @@ func (r *Redis) FindDriverIDs(ctx context.Context, location string) []string {
 	for i := range drivers {
 		drivers[i] = fmt.Sprintf("T7%05dC", rand.Int()%100000)
 	}
+	msg := fmt.Sprintf("FindDriverIDs(%s) = [%s]", location, strings.Join(drivers, ", "))
+	r.logger.For(ctx).Info(msg)
```
Comment (yurishkuro) [inline examples/hotrod/services/driver/redis.go]: Zap has a field for arrays: https://github.com/uber-go/zap/blob/master/array.go#L98

```
r.logger.For(ctx).Info("Found drivers", zap.Strings("drivers", drivers))
```

Comment (isaachier) [reply examples/hotrod/services/driver/redis.go]: Much cleaner thanks. Don't know zap too well.

**Pull request merged**